# expressed as * 1E7
int32 Latitude
# expressed as * 1E7
int32 Longitude
# expressed as * 1000 (millimeters)
int32 AltitudeRelative
# (-pi...pi)
float32 Yawangle
float32 vx
float32 vy
float32 vz
float32 vyaw
#reference abs or rel
int8 Mode
# coordinate frame
uint8 frame